internals/constants/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "1.0.4",
   "types": "dist/index.d.ts",
   "main": "dist/index.js",
+  "files": [
```

Comment:
This is a private package. Why do we need to specify the `files`?

Reply:
Its good practice. Depending on the package manager, it might change the behavior of workspaces so you don't have access to files outside of the allowlist. I haven't confirmed this though.